Signup login

ewynx opened the pull request:
The initial screen is now a decision screen which leads to either signup or login.

This is very minimal and the signup screen lacks some info about the privacy, because I imagine people can worry about that and it is a positive point of the app!

After the Signup screen I want to add a little introduction - now it just goes into the questions right away, but I'll make a separate branch for that.

ewynx committed:
According to design: the initial screen with the decision point of Login/Signup.

ewynx committed:
Adjusted login screen.

ewynx committed:
A simple sign up screen

ewynx committed:
Cleanup

peterbertel commented on i18n/en.json:
```diff
@@ -66,5 +66,10 @@
+  "go back": "Go back",
+  "Sign up": "Sign up",
+  "enter name": "Please enter your name",
+  "name": "Name"
```
Do you think we need to make similar edits to `es.json` and `nl.json`? I haven't tested running the app with those other languages, so I'm not sure if the app will crash if those values are missing.

ewynx replied:
You're very correct! Thanks

peterbertel committed:
Rename the first login page to 'splash-screen' and navigate to that page upon logging out

peterbertel commented:
@ewynx I made a couple changes that I think improve the logout experience. Before, the app would take the user to the `login-page-2` screen when clicking the `Log out` button in the settings page. From the `login-page-2` screen, the user was able to click the `Go back` button and navigate back to the app settings page, presumably logging the user back in. To me, I think it feels confusing to be able to log back in by clicking the `Go back` button after logging out. To simplify things, I changed the `Log out` button to send the user to the `login-page-1` screen, which I renamed as the `splash-screen`. Let me know what you think.

ewynx committed:
PR comment fix: Added translations.

ewynx committed:
- There's a bug with the translation on login_screen_2
- On the Log in/Create account screen going back should be disabled now. So when you logout and get to this screen, you can't press back (on Android) and be logged in again.

ewynx commented:
@peterbertel I like it. On the Log in/Create account screen (that you named splash screen) I disabled the back button, which should add on to that more logical flow for logging out/logging in. So when you're logged out you go to the splash screen and can't get back in the app by clicking on the back button (Android).

peterbertel commented:
@ewynx Good call disabling the back button for Android. I think the changes are ready to merge!

Feel free to merge when you are ready.
